src/App.js

```diff
@@ -15,17 +15,10 @@ function App() {
           className={ !isShown ? "App-logo" : "App-logo-reversed" }
           alt="logo"
         />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
+        <p style={{textAlign: "center"}}>
+          <h1>Welcome to the hell marine!</h1>
```

Comment:
Please add the transparent gray box for all the content to improve readability

Reply:
Made another solution and resolved this problem succsessfully.